build.gradle

```diff
@@ -75,6 +75,9 @@ dependencies {
     // the conversion to batik is complete.
     implementation files('lib/svgSalamander.jar')
 
+    // Light and Dark themes
+    implementation 'com.formdev:flatlaf:0.26'
```

Comment:
Dependencies are transitive, so this is redundant because it's inherited from the MM dependency.